vms/job/tests/test_searchJob.py

```diff
@@ -0,0 +1,433 @@
+# third party
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+
+# Django
+from django.contrib.staticfiles.testing import LiveServerTestCase
+
+# local Django
+from job.services import get_jobs_by_date
+from pom.pages.authenticationPage import AuthenticationPage
+from pom.pages.jobSearchPage import JobSearchPage
+from shift.utils import (create_admin, create_volunteer_with_details,  create_organization_with_details, register_event_utility, create_job_with_details, create_shift_with_details)
+
+
+class SearchJob(LiveServerTestCase):
+    """
+    SearchEvent class contains tests to check '/event/search/' view.
+    Choices of parameters contains
+    - Name
+    - Start Date
+    - End Date
+    - City
+    - State
+    - Country
+    - Event
+    Class contains 7 tests to check each parameter separately and also to check
+    if a combination of parameters entered, then intersection of all results is
+    obtained.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        cls.driver = webdriver.Firefox()
+        cls.driver.implicitly_wait(5)
+        cls.driver.maximize_window()
+        cls.search_page = JobSearchPage(cls.driver)
+        cls.authentication_page = AuthenticationPage(cls.driver)
+        cls.wait = WebDriverWait(cls.driver, 10)
+        super(SearchEvent, cls).setUpClass()
```

Comment:
`super(SearchEvent, cls).setUpClass()` should be `super(SearchJob, cls).setUpClass()`